### PR TITLE
improve test stability

### PR DESCRIPTION
**Pull request by wilsonwang371**
## Why are these changes needed?

1. improve compatibility test stability by increasing the initial wait time in liveness/readiness probe.
2. remove unnecessary gofumpt run in Makefile.

## Related issue number

N/A

## Checks

- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [X] Manual tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by Jeffwan** [comment]
What's the reason the test takes that long to finish?

**Comment by wilsonwang371** [comment]
@Jeffwan I did some investigation, it looks like the root cause for the failure and long waiting time is because in our test environment, the initial delay seconds is too small.  I am working on this right now.

File: ray-operator/Makefile [hunk 1]
```diff
@@ -59,14 +59,14 @@ fumpt: gofumpt
 	$(GOFUMPT) -l -w ../
 
 ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
-test: manifests generate fmt vet fumpt ## Run tests.
```
Comment (Jeffwan) [inline ray-operator/Makefile:62]: em. Why do revert the change here?

Comment (wilsonwang371) [reply ray-operator/Makefile:62]: this is some extra checks which we already done in our github flow.

Comment (Jeffwan) [reply ray-operator/Makefile:62]: `make gofumpt` so we suggest user to manually run this if there're CI errors? that sounds fine.

**Comment by Jeffwan** [comment]
Currently seems it has two issues
1. flaky and unstable issue - sometimes succeed and sometimes failed
2. takes long time to finish

If we can resolve the flake issue and reduce running time to 10mins, that would be great.

File: ray-operator/Makefile (hunk 1 above)
Comment (wilsonwang371) [reply ray-operator/Makefile:62]: yes

**Comment by wilsonwang371** [comment]
From what i observed, when we killed the head node in a ray cluster in our github testbed, the worker readiness probe can fail for more than 30 seconds. This will lead to our default readiness probe getting involved and kill the worker node.

I increased the readiness probe timeout to 40 seconds for now.

FYI: @iycheng @brucez-anyscale @Jeffwan @scarlet25151 @DmitriGekhtman

**Review by Jeffwan — APPROVED** [review]

**Comment by brucez-anyscale** [comment]
@iycheng I think the worker node readiness or liveness do not depend on head node. Because if worker node cannot connect head node, it will return healthy.
Can you confirm?

**Comment by iycheng** [comment]
Yes. Tested here https://github.com/ray-project/ray/blob/master/dashboard/modules/healthz/tests/test_healthz.py#L32

**Comment by brucez-anyscale** [comment]
@wilsonwang371 Can you confirm the worker issue is caused by head node down?

**Comment by wilsonwang371** [comment]
> @wilsonwang371 Can you confirm the worker issue is caused by head node down?

this is due to probe timeout is not big enough for kind cluster
